README.md

```diff
@@ -76,22 +76,29 @@ look like this:
   - test/
 ```
 
-`pulp` works by convention. It expects all projects to contain a
-`bower.json` file, which is needed for package management (which, in
-PureScript, is handled by [Bower](http://bower.io/)). `pulp` itself
-currently doesn't use this file for anything other than figuring out
-where the project root is.
-
-Your project source files go in the `src` folder. Your test files go
-in the `test` folder. Project dependencies will be installed under the
-Bower standard `bower_components` folder, and are expected to have the
-same basic `src`/`test` structure. That's all there is to a `pulp` project.
+`pulp` works by convention. Your project source files go in the `src`
+folder. Your test files go in the `test` folder. By default
+[Bower](http://bower.io/) is used for package management, but you can
+also use [`psc-package`](#psc-package). Project dependencies will be
+installed under the `bower_components` or `.psc-package` folder,
+depending on which package manager you use, and are expected to have
+the same basic `src`/`test` structure. That's all there is to a `pulp`
+project.
 
 We employ the `purescript-` prefix as a convention to identify PureScript
 projects when they're used as dependencies. You're welcome to call your
 project anything you like, but without the `purescript-` prefix it won't
 be picked up by `pulp` as a dependency when installed through Bower.
 
+### `psc-package`
+
+`pulp` has support for using `psc-package`, an experimental package
+manager for PureScript based on the concept of package sets, instead of
+Bower. To start a project using `psc-package` use the `--psc-package`
+flag when initializing a project: `pulp init --psc-package`. Refer to
+[the `psc-package` readme](https://github.com/purescript/psc-package#psc-package)
+for details on how to manage packages with `psc-package`.
```

Comment:
Am I correct in thinking that after this is merged, pulp will use psc-package if a psc-package.json file exists, and otherwise it will use Bower? I think it would be good to explain it this way here, and then afterwards mention what `pulp init --psc-package` does, because there are other ways of creating psc-package.json files in addition to `pulp init --psc-package`, and people who have used those might be surprised by pulp's behaviour.